__tests__/app.test.js

```diff
@@ -75,6 +75,15 @@ describe("/api/articles/:article_id", () => {
         );
       });
   });
+  test("GET 200: the same as the test before adding comment_count", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.comment_count).toBe("11");
```

Comment:
great job doing a new test for this - I would pop the other properties on here too